java_src/foam/nanos/pm/DAOPMLogger.java

```diff
@@ -7,46 +7,48 @@
 package foam.nanos.pm;
 
 import foam.core.ContextAwareSupport;
+import foam.dao.DAO;
 import foam.dao.MapDAO;
+import foam.dao.ProxyDAO;
 import foam.nanos.NanoService;
 
 public class DAOPMLogger
-  extends    ContextAwareSupport
+  extends    ProxyDAO
```

Comment:
This shouldn't extend ProxyDAO. It isn't a DAO, it's a PMLogger. Find the pmInfoDAO in the context and use that for your DAO.